consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -202,7 +202,9 @@ public void run() {
             findRequest.where(and(
                     withValue("configurationName = " + configuration.getConfigurationName()),
                     withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    not(withSubfield("jobExecutions", withValue("completedFlag = true")))));
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_SUCCESS'"))),
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_PARTIAL'")))
```

Comment:
The getMigrationJobs function should not return jobs that are currently being executed, but needs to pick up jobs that were marked for execution a long time ago.  So, a job can be returned by this function when:

```
and:
    whenAvailableDate <= new Date()
    not:
        jobExecutions.jobStatus $in ['COMPLETED_SUCCESS', 'COMPLETED_PARTIAL']
    and:
        not: 
            jobExecutions.jobStatus $not_in [COMPLETED_SUCCESS', 'COMPLETED_PARTIAL']
        jobExecutions.actualStartDate < (new Date() - expectedExecutionMilliseconds)
```

but please check my logic here.  We want to pickup work that has been marked in progress for too long.